src/app/OperationalSessionSetup.h

```diff
@@ -305,6 +313,8 @@ class DLL_EXPORT OperationalSessionSetup : public SessionEstablishmentDelegate,
 
     bool mPerformingAddressUpdate = false;
 
+    TransportPayloadCapability mSessionForLargePayload = TransportPayloadCapability::kDefault;
```

Comment:
mSessionForLargePayload does not reflect the meaning of TransportPayloadCapability well, maybe mTransportPayloadCapability or  mIsSessionForLargePayload